vault/logical_system.go

```diff
@@ -5680,6 +5722,15 @@ This path responds to the following HTTP methods.
 			Returns a list of entries in specified table
 		`,
 	},
+	"internal-inspect-request": {
```

Comment:
Cosmetic: is there a way to avoid the repetition "Information on the information" ?